Viewer/src/configuration/interfaces/environmentMapConfiguration.ts

```diff
@@ -0,0 +1,16 @@
+export interface IEnvironmentMapConfiguration {
+    /**
+     * Environment map texture path in relative to the asset folder.
+     */
+    texture: string;
+
+    /**
+     * Default rotation to apply to the environment map.
+     */
+    rotationY: number;
+
+    /**
+     * Tint level of the main color on the environment map.
+     */
+    tintLevel: number;

```

Comment:
There is a tint level but is there any tint color ?

Reply:
There is main color, I should move it here instead of in scene. Will do that